server/src/main/java/com/tencent/rss/server/ShuffleServer.java

```diff
@@ -138,6 +146,17 @@ private void initialization() throws Exception {
       multiStorageManager = new MultiStorageManager(shuffleServerConf, id);
       multiStorageManager.start();
     }
+
+    boolean healthCheckEnable = shuffleServerConf.getBoolean(ShuffleServerConf.RSS_HEALTH_CHECK_ENABLE);
```

Comment:
Could you use storage type to enable specific checker, since you already consider mulitple checkers scenario.

Reply:
Another configuration `RSS_HEALTH_CHECKER_CLASS_NAME`

Reply:
The *health check* should not be coupled with storage type. We could disable some checker if condition not matched (eg, storage type)

Reply:
Too complex.

Reply:
HealthCheck's level and scope is higher than a specific disk usage checker, which is part of the HealthCheck. And It's not appropriate to couple HealthCheck with storage type. We could use more simple way, using conf when start checker.

Reply:
OK.